src/renderer/components/MainSideBar/index.js

```diff
@@ -332,29 +333,29 @@ const MainSideBar = () => {
                 disabled={noAccounts}
                 collapsed={secondAnim}
               />
-              {/* TODO Make this toggleable via an experimental flag/setting */}
-              <SideBarListItem
-                id={"swap"}
-                label={t("sidebar.swap")}
-                icon={IconSwap}
-                iconActiveColor="wallet"
-                onClick={handleClickSwap}
-                isActive={
-                  location.pathname.startsWith("/swap") && !location.pathname.startsWith("/swap2")
-                }
-                disabled={noAccounts}
-                collapsed={secondAnim}
-              />
-              <SideBarListItem
-                id={"swap2"}
-                label={t("sidebar.swap2")}
-                icon={IconSwap}
-                iconActiveColor="wallet"
-                onClick={handleClickSwap2}
-                isActive={location.pathname.startsWith("/swap2")}
-                disabled={noAccounts}
-                collapsed={secondAnim}
-              />
+              {isSwapV2Enabled ? (
```

Comment:
Just leave it as a single swap entry, same wording and same route regardless of the flag